docs/contributing/provider_development_guide.md

```diff
@@ -70,6 +74,12 @@ language with the Crossplane community. This guide targets [crossplane-runtime
 v0.9.0]. It assumes the reader is familiar with the Kubernetes [API Conventions]
 and the [kubebuilder book].
 
+> If you are building a new provider from scratch, instead of adding new
+resources to an already existing one, please use [provider-template] repository
+as a template by hitting the `Use this template` button in GitHub UI. It
+codifies most of the best practices used by the Crossplane community so far and
+is the easiest way to start a new provider.
+
 ## Defining Resource Kinds
```

Comment:
This isn't how we add new kinds anymore, I think it should talk about copy-paste'ing an existing one instead of calling kubebuilder.

Reply:
Agreed, but I would prefer not to extend scope of this work because it would end up more changes than just removing that part. I feel like we would need to rewrite this guide step by step starting with provider-template, similar to https://github.com/crossplane/terrajet/blob/main/docs/generating-a-provider.md#generate.